lib/proc/proc_tokenize.c

```diff
@@ -54,6 +54,29 @@ fin_tokens(grn_ctx *ctx, grn_obj *tokens)
   GRN_OBJ_FIN(ctx, tokens);
 }
 
+static void
+output_tokens_simple(grn_ctx *ctx, grn_obj *tokens, grn_obj *lexicon)
+{
+  size_t i;
+  size_t n_tokens = GRN_BULK_VSIZE(tokens) / sizeof(tokenize_token);
+
+  grn_ctx_output_array_open(ctx, "TOKEN_VALUES", (int)n_tokens);
+  grn_obj key;
+  GRN_OBJ_INIT(&key, GRN_BULK, GRN_OBJ_DO_SHALLOW_COPY, lexicon->header.domain);
+  for (i = 0; i < n_tokens; i++) {
+    tokenize_token *token = ((tokenize_token *)(GRN_BULK_HEAD(tokens))) + i;
```

Comment:
How about defining `tokenize_token *tokenize_tokens = (tokenize_token *)GRN_BULK_HEAD(tokens);` out of `for` and use it in `for` like `tokenize_token *token = tokenize_tokens + i`?

Reply:
fix: [cbbb099](https://github.com/groonga/groonga/pull/2302/commits/cbbb099920e12ac13ec7ac02fedf57c952501b37) Thanks, it looks better.